redash/models/__init__.py

```diff
@@ -77,6 +77,7 @@ class TableMetadata(TimestampMixin, db.Model):
     description = Column(db.String(4096), nullable=True)
     column_metadata = Column(db.Boolean, default=False)
     sample_query = Column("sample_query", db.Text, nullable=True)
+    latest_sampling = Column(db.DateTime(True), nullable=True)
```

Comment:
It's a great practice to indicate column return type in the naming and we've settled on the convention to use `<something>_at` for the naming. So instead of implying that `latest_sampling` points to THE latest samply let's use `sample_query_updated_at` or just `sample_updated_at` if you prefer.